app/main.py

```diff
@@ -1,25 +1,65 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.name = '_' + name
+
+    def __get__(self, instance: Any, owner: str) -> Any:
+        return getattr(instance, self.name, None)
+
+    def __set__(self, instance: Any, value: Any) -> Any:
+        if not isinstance(value, int):
+            raise TypeError("Value must be an integer")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError("Value out of range")
+        setattr(instance, self.name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
```

Comment:
The `SlideLimitationValidator` class inherits from `ABC`, but it does not define any abstract methods. Consider removing the inheritance from `ABC` unless you plan to add abstract methods.